frontend/src/api/k8s/notebooks.ts

```diff
@@ -93,11 +95,13 @@ export const assembleNotebook = (
     volumeMounts.push(getshmVolumeMount());
   }
 
-  const hardwareProfileNamespace: Record<string, string | null> = selectedHardwareProfile
-    ? selectedHardwareProfile.metadata.namespace === projectName
-      ? { 'opendatahub.io/hardware-profile-namespace': projectName }
-      : { 'opendatahub.io/hardware-profile-namespace': dashboardNamespace }
-    : { 'opendatahub.io/hardware-profile-namespace': null };
+  const isAcceleratorProfileSelected = !!selectedAcceleratorProfile;
+  const hardwareProfileNamespace: Record<string, string | null> =
+    selectedHardwareProfile && isAcceleratorProfileSelected
```

Comment:
Is there a reason you're checking that both hardware profiles and accelerator profiles are selected?

Reply:
that was a bug i fixed; it's now `  selectedHardwareProfile && !isAcceleratorProfileSelected`